core/Stream/hsStream.h

```diff
@@ -87,7 +87,11 @@ class HSPLASMA_EXPORT hsStream
 
 class HSPLASMA_EXPORT hsFileStream : public hsStream {
 protected:
+#ifdef WIN32
+    HANDLE F;
```

Comment:
I don't love having a `HANDLE` in a header file, especially since we've managed to avoid it everywhere else...  Perhaps we could just store a generic `void* F` and then cast it as necessary within `hsStream.cpp`?

Reply:
It looks like MSVC is happy to accept a `void*` without casting any time a function wants a `HANDLE`. That's nice.